signer/contentsignaturepki/contentsignature.go

```diff
@@ -84,10 +85,14 @@ func New(conf signer.Configuration) (s *ContentSigner, err error) {
 	if conf.ID == "" {
 		return nil, fmt.Errorf("contentsignaturepki %q: missing signer ID in signer configuration", s.ID)
 	}
-	if conf.PrivateKey == "" {
-		return nil, fmt.Errorf("contentsignaturepki %q: missing private key in signer configuration", s.ID)
+	if conf.IssuerPrivKey == "" {
+		return nil, fmt.Errorf("contentsignaturepki %q: missing issuer private key in signer configuration", s.ID)
 	}
-	s.issuerPriv, s.issuerPub, s.rand, _, err = conf.GetKeysAndRand()
+	// we need to parse or retrieve from the hsm the issuer private key,
```

Comment:
A handle for the issuer private key, right? We shouldn't require it to be exportable to the autograph CU.